polkadot-host-spec/c03-transition.tm

```diff
@@ -670,112 +671,305 @@
+  32-bit integers:
+
+  <\itemize-dot>
+    <item><verbatim|interval> - The interval (in blocks) at which those pairs
+    should be created. If this value is less or equal to 1 it means that
+    those pairs are not created at all.
+
+    <item><verbatim|levels> - The maximum number of \Plevels\Q in the
+    hierarchy. If this value is 0 it means that those pairs are not created
+    at all.
+  </itemize-dot>
+
+  For each level from 1 to <verbatim|levels>, the Polkadot Host creates those
+  pairs on every <verbatim|<math|<text|interval<rsup|level><verbatim|>>>>-nth
+  block, formally applied as:
+
+  <\algorithm|<name|Key-To-Block-Pairs>(<math|B<rsub|i>>, interval, levels)>
+    <strong|Initialize:>
+
+    1. <math|I=interval>
+
+    2. <math|L=levels>
+
+    <strong|for each> <math|l\<in\><around*|{|1,\<ldots\>,L|}>>
+
+    3.<space|1em>if <math|H<rsub|i><around*|(|B<rsub|i>|)>=I<rsup|l>>
+
+    4.<space|2em><name|Insert-Blocks>(<math|H<rsub|i><around*|(|B<rsub|i>|)>>,
+    <math|I<rsup|l>>)
+  </algorithm>
+
+  <\itemize-dot>
+    <item><math|B<rsub|i>> implies the block at which those pairs gets
+    inserted into the Changes Trie.
+
+    <item><name|Insert-Blocks> - Inserts every block number within the range
+    <math|H<rsub|i><around*|(|B<rsub|i>|)>-I<rsup|l>+1> to
+    <math|H<rsub|i><around*|(|B<rsub|i>|)>> in which any extrinsic changed
+    the specified key.
+  </itemize-dot>
+
+  \;
+
+  For example, let's say <verbatim|interval> is set at <verbatim|4> and
+  <verbatim|levels> is set at <verbatim|3>. This means there are now three
+  levels which get generated at three different occurences:
+
+  <\enumerate-numeric>
+    <item><strong|Level 1> - Those pairs are generated at every
+    <math|<text|<strong|4<rsup|1>>>>-nth block, where the pair value contains
+    the block numbers of every block that changed the specified storage key.
+    This level only considers block numbers of the last four
+    (<math|=4<rsup|1>>) blocks.
+
+    <\itemize-dot>
+      <item>Example: this level occurs at block 4, 8, 12, 16, 32, etc.
+    </itemize-dot>
+
+    <item><strong|Level 2> - Those pairs are generated at every
+    <math|<text|<strong|4<rsup|2>>>>-nth block, where the pair value contains
+    the block numbers of every block that changed the specified storage key.
+    This level only considers block numbers of the last 16
+    (<math|=4<rsup|2>>) blocks.
+
+    <\itemize-dot>
+      <item>Example: this level occurs at block 16, 32, 64, 128, 256, etc.
+    </itemize-dot>
+
+    <item><strong|Level 3> - Those pairs are generated at every
+    <text|<math|<text|<strong|4<rsup|3>>>>>-nth block, where the pair value
+    contains the block numbers of every block that changed the specified
+    storage key. this level only considers block number of the last 64
+    (<math|=4<rsup|3>>) blocks.
+
+    <\itemize-dot>
+      <item>Example: this level occurs at block 64, 128, 196, 256, 320, etc.
+    </itemize-dot>
+  </enumerate-numeric>
+
```

Comment:
in my opinion there are too many example here. You might one to drop at least one. But your call.
I would also demonstrate the whole calculation for at least one of the example the exact value for the key (type, ?, Key) and the C_Value/A_value.